test_postgresql.sh

```diff
@@ -0,0 +1,3 @@
+#! /usr/bin/env bash
```

Comment:
`#!/bin/bash` is conventional for shell scripts.

Does it need a `set -e` so that it doesn't plough on with the `run` if the `build` fails?